lib/typescript/httpclient/index.ts

```diff
@@ -16,21 +16,12 @@ import {
   ConversationPayload,
   ChannelPayload,
   ChannelsPayload,
+  UserPayload,
 } from './payload';
-import {
-  connectChannelMapper,
-  channelMapper,
-  channelsMapper,
-  disconnectChannelApiMapper,
-  conversationsMapper,
-  messageMapperData,
-  tagsMapper,
-  userMapper,
-  paginatedPayloadMapper,
-  messageMapper,
-  conversationMapper,
-} from './mappers';
-import {TagColor, Tag} from './model';
+
+import {TagColor, Tag, Message} from './model';
+/* eslint-disable @typescript-eslint/no-var-requires */
+const camelcaseKeys = require('camelcase-keys');
```

Comment:
here we have to use _require_ and not _import__ because the library supports TS but is not in TS